tests/test_configuration.py

```diff
@@ -21,12 +33,12 @@
     'module-bays',
 )
 
-ROOT_DIR = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '..'))
+ROOT_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
 
 KNOWN_SLUGS = set()
 KNOWN_MODULES = set()
 
-USE_LOCAL_KNOWN_SLUGS = False
-USE_UPSTREAM_DIFF = True
+USE_LOCAL_KNOWN_SLUGS = os.environ.get('USE_LOCAL_KNOWN_SLUGS', False)
```

Comment:
Doesn't this need to also be passed to `distutils.util`s `strtobool` function? Environment variables are going to be strings.